en/02_quicktour.md

```diff
@@ -0,0 +1,276 @@
+
+    SQL("insert into members (name, birthday, created_at) values ({name}, {birthday}, {createdAt})")
+      .bindByName('name -> name, 'birthday -> None, 'createdAt -> createdAt)
+      .update.apply()
+
+as well as the executable template where you write binding variables in the comments accompanied by dummy values;
+
+    SQL("""
+      insert into members (name, birthday, created_at) values (
+        /*'name*/'Alice', 
+        /*'birthday*/'1980-01-01', 
+        /*'createdAt*/current_timestamp
+      )
+      """)
+      .bindByName('name -> name, 'birthday -> None, 'createdAt -> createdAt)
+      .update.apply()
+
+This is explained in more detail at the SQL template section in detail.
+
+Let's return to the example and issue the same `select` statement once again.
+
+    val members: List[Map[String, Any]] = DB readOnly { implicit session =>
+      SQL("select * from members").map(_.toMap).list.apply()
+    }
+    // => members: List[Map[String,Any]] = List(Map(ID -> 1, NAME -> Alice, BIRTHDAY -> 1980-01-01, CREATED_AT -> 2012-12-31 00:02:09.247), Map(ID -> 2, NAME -> Bob, CREATED_AT -> 2012-12-31 00:02:09.247))
+
+The two records that was inserted has been returned as expected. You find that the previous `insert` was processed successfully.
+
+Previously in the `select` examples, we obtained our results as `Map[String, Any]`, but let's change it so that the results are mapped to a `Member` class.
+
+ScalikeJDBC doesn't require you to do any special configuration to the class mapped from `ResultSet`. It is OK to simply define it as a `case class` or just as a regular `class`.
+
+It is recommended to define `NOT NULL` columns as `Option` types and use DateTime and LocalDate from [Joda Time](http://www.joda.org/joda-time/) for date and timestamp columns. The Date Time API of Java SE 8 can be used as well, but I will explain it separately. In this sample, I will show you an example of using Joda Time.
+
+    case class Member(
+      id: Long, 
+      name: String, 
+      description: Option[String] = None, 
+      birthday: Option[LocalDate] = None, 
+      createdAt: DateTime)
+    
+    val allColumns = (rs: WrappedResultSet) => Member(
+      id = rs.long("id"), 
+      name = rs.string("name"), 
+      description = rs.stringOpt("description"),
+      birthday = rs.dateOpt("birthday").map(_.toLocalDate), 
+      createdAt = rs.jodaDateTime("created_at")
+    )
+    
+    val members: List[Member] = DB readOnly { implicit session =>
+      SQL("select * from members limit 10").map(allColumns).list.apply()
+    }
+    // => members: List[Member] = List(Member(1,Alice,None,Some(1980-01-01),2012-12-31T00:02:09.247+09:00), Member(2,Bob,None,None,2012-12-31T00:02:09.247+09:00))
+
+### SQL Interpolation
+
+Since [String Interpolation (SIP-11)](http://docs.scala-lang.org/sips/pending/string-interpolation.html) was introduced to Scala 2.10.0, you can embed expressions to strings by `${...}`.
```

Comment:
s/introduced to Scala 2.10.0/introduced in Scala 2.10.0/

[String Interpolation (SIP-11)](http://docs.scala-lang.org/sips/pending/string-interpolation.html) introduced in Scala 2.10.0 enables you embedding expressions surrounded by `${...}` into String values.